compiler/bindings/c/iree/compiler/dialects/iree_codegen.h

```diff
@@ -7,6 +7,7 @@
 #ifndef IREE_COMPILER_DIALECTS_IREE_CODEGEN_H
 #define IREE_COMPILER_DIALECTS_IREE_CODEGEN_H
 
+#include <cstdint>
```

Comment:
I will delete this before landing; clangd keeps re-adding it because it indexes the file as c++...